documentcloud/documents/views.py

```diff
@@ -434,6 +443,10 @@ def _run_addons(self, document, old_processing):
             for event in events:
                 event.dispatch(document_pk=document.pk)
 
+    def _set_page_text(self, document, validated_data):
```

Comment:
Yeah, that makes sense - I think I just did that since the other similar functions needed `validated_data` for one reason or another, but this one could just take pages